docs/uploader/aws-s3-multipart.mdx

```diff
@@ -410,8 +410,83 @@ uppy.use(AwsS3Multipart, {
 });
 ```
 
+#### `getTemporarySecurityCredentials(options)`
```

Comment:
perhaps an extra `:::note` that links to the example in the uppy repo?

Reply:
Currently there are no examples that uses that option, so I guess this should wait until there's one (which cannot happen until that option ships into a release, because we are using the CDN in our demo)